coalib/bears/requirements/GoRequirement.py

```diff
@@ -29,3 +30,23 @@ def __init__(self, package, version="", flag=""):
         """
         PackageRequirement.__init__(self, 'go', package, version)
         self.flag = flag
+
+    def install_command(self):
+        """
+       Creates the installation command for the instance of the class.
+
+       >>> GoRequirement
+       ('github.com/golang/lint/golint', '' , '-u' ).install_command()
+       'go get -u github.com/golang/lint/golint'
+
+       :param return: A string with the installation command.
+       """
+        return "go get {} {}".format(self.flag, self.package)
+
+    def is_installed(self):
+        """
+       Checks if the dependency is installed.
+
+       :param return: True if dependency is installed, false otherwise.
+       """
+        return not call_without_output(('go', 'show', self.package),)
```

Comment:
i tried ``go show`` on my machine, but says ``theres no such thing as "show"`` for go, have you tested this .. ?